migrations/20230615025700_create_ibc_progress_table.ts

```diff
@@ -0,0 +1,18 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('ibc_progress', (table) => {
```

Comment:
cái bảng nối này có vẻ hơi to, có cách lưu khác ko? index mấy cái hash cũng hơi nhiều